Mlem/App/Globals/Definitions/AccountsTracker.swift

```diff
@@ -25,11 +25,24 @@ class AccountsTracker {
     
     var userAccounts: [UserAccount] = .init()
     var guestAccounts: [GuestAccount] = .init()
-    var defaultAccount: any Account { userAccounts.first ?? defaultGuestAccount }
+    
+    // Used on startup to determine which account should be made active
+    func mostRecentAccount() -> any Account {
+        var allAccounts: [any Account] = userAccounts + guestAccounts
+        if let activeAccount = allAccounts.first(where: { $0.activityState == .active }) {
+            return activeAccount
+        }
+        let sorted = allAccounts.sorted(by: { $0.activityState.lastUsed ?? .distantPast < $1.activityState.lastUsed ?? .distantPast })
+        if let lastUsedAccount = sorted.last {
+            return lastUsedAccount
+        }
+        return defaultGuestAccount
```

Comment:
This should be the first user account, if present--otherwise the app puts you on a guest account at first launch following this patch
```suggestion
        return userAccounts.first() ?? defaultGuestAccount
```